### PR TITLE
arduino: update to 1.8.8.

### PR DESCRIPTION
**Pull request by Gigahawk**

### CODE REVIEW

File: srcpkgs/arduino/template [hunk 1]
```diff
@@ -11,8 +11,10 @@ homepage="http://arduino.cc/"
 distfiles="
  https://github.com/arduino/Arduino/archive/${version}.tar.gz
  ${SOURCEFORGE_SITE}/astyle/astyle_3.0.1_linux.tar.gz"
-checksum="57e8c4cf9147135a10cfd45431189a675b1c39ddf833eaeed553f4fc869fdf69
+ checksum="01bf72e54165a522fe80ae5454b7073994fd4139a326f2edc0bb61b1d226179a
```
Comment (maxice8) [inline srcpkgs/arduino/template]: remove extra whitespace

File: srcpkgs/arduino/template [hunk 2]
```diff
@@ -11,8 +11,10 @@ homepage="http://arduino.cc/"
...
  6c3ab029e0e4a75e2e603d449014374aa8269218fdd03a4aaa46ab743b1912fd"
+
```
Comment (maxice8) [inline srcpkgs/arduino/template]: remove 1 extra blank line

File: srcpkgs/arduino/template [hunk 3]
```diff
@@ -11,8 +11,10 @@ homepage="http://arduino.cc/"
...
+
 nostrip=yes
 wrksrc=Arduino-$version
```
Comment (maxice8) [inline srcpkgs/arduino/template]: move wrksrc= to  directly above hostmakedepends

**Comment by maxice8** [comment]
ping

**Comment by Gigahawk** [comment]
Sorry for the long delay, got busy with school and stuff.

According to [this](https://github.com/arduino/Arduino/issues/8255), it looks like 1.8.8 has to be built with the `oracle-jdk`, but if you look into the [build logs](https://travis-ci.org/void-linux/void-packages/jobs/484172400) it looks like there's something wrong with `oracle-jdk` at the moment?

I get the same error when trying to build locally.

**Comment by maxice8** [comment]
Repos should be fixed now but there is an error due to the usage of oracle-jdk instead of openjdk

**Comment by jnbr** [comment]
#9186

**Pull request closed without merging**